Sources/Routing/Protocols/NavigationPathManageable.swift

```diff
@@ -10,28 +10,27 @@ import SwiftUI
 public protocol NavigationPathManageable: ObservableObject {
```

Comment:
```suggestion
public protocol DestinationManageable: ObservableObject {
```

Does it make sense to rename this now that it isn't managing a navigation path?